.github/workflows/nightly.yaml

```diff
@@ -0,0 +1,50 @@
+name: Nightly-build
+
+on:
+  schedule:
+    - cron: '0 2 * * *'
+
+jobs:
+  push-images:
+    env:
+      working-directory: ./ray-operator
+      cli-working-directory: ./cli
+    name: Push Nightly Images
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Set up Go 1.17.x
+      uses: actions/setup-go@v2
+      with:
+        # Use the same go version with build job
+        go-version: '1.17'
+
+    - name: Check out code into the Go module directory
+      uses: actions/checkout@v2
+      with:
+        ref: master
+
+    - name: Get dependencies
+      working-directory: ${{env.working-directory}}
+      run: go mod download
+
+    - name: Set up Docker
+      uses: docker-practice/actions-setup-docker@master
+
+    - name: install kubebuilder
+      run: |
+        wget https://github.com/kubernetes-sigs/kubebuilder/releases/download/v3.0.0/kubebuilder_$(go env GOOS)_$(go env GOARCH)
+        sudo mv kubebuilder_$(go env GOOS)_$(go env GOARCH) /usr/local/bin/kubebuilder
+
+    - name: Build Docker Image
+      run: IMG=kuberay/operator:nightly make docker-image
```

Comment:
I kind of forget whether we need kubebuilder to make docker-image?

Reply:
I think we can remove kubebuilder here.